src/dashboard/apps/consent/tests/test_views.py

```diff
@@ -0,0 +1,146 @@
+"""Dashboard consent views tests."""
+
+import pytest
+from django.urls import reverse
+
+from apps.auth.factories import UserFactory
+from apps.consent import AWAITING, VALIDATED
+from apps.consent.factories import ConsentFactory
+from apps.consent.models import Consent
+from apps.consent.views import ConsentFormView
+from apps.core.factories import DeliveryPointFactory, EntityFactory
+from apps.core.models import DeliveryPoint
+
+
+@pytest.mark.django_db
+def test_bulk_update_consent_status_without_ids(rf):
+    """Test that no status is updated if no id is passed."""
+    request = rf.get(reverse("consent:manage"))
+    request.user = UserFactory()
+
+    view = ConsentFormView()
+    view.setup(request)
+
+    ConsentFactory.create_batch(4)
+
+    ids = []
+    assert view._bulk_update_consent(ids, VALIDATED) == 0
+    for consent in Consent.objects.all():
+        assert consent.status == AWAITING
```

Comment:
We usually check data before and after using a method that can modify it:

```suggestion
    size = 4
    ConsentFactory.create_batch(size)

    assert all(c == AWAITING for c in Consent.objects.values_list("status", flat=True))
    assert view._bulk_update_consent([], VALIDATED) == 0
    assert all(c == AWAITING for c in Consent.objects.values_list("status", flat=True))
```